src/schemas/json/sarif-1.0.0.json

```diff
@@ -386,19 +402,18 @@
 
         "fileName": {
           "description": "The fully qualified path to the analysis tool.",
-          "type": "string",
-          "default": { }
+          "type": "string"
         },
 
         "workingDirectory": {
           "description": "The working directory for the analysis rool run.",
-          "type": "string",
-          "default": { }
+          "type": "string"
         },
 
         "environmentVariables": {
           "description": "The environment variables associated with the analysis tool process, expressed as key/value pairs.",
           "type": "object",
+          "additionalProperties": true,
           "default": { }
```

Comment:
Looks reasonable. It's an empty object. But note that our generated code doesn't respect the default, and if we deserialize a SARIF file that doesn't specify invocation.environmentVariables, the value of the property Invocation.EnvironmentVariables will be null, not an empty dictionary.

This is an existing bug (the only change I made here was to explicitly specify `additionalProperties` on the line above. I filed Microsoft/sarif-sdk#424, "Respect schema defaults".